.github/workflows/fork-ci.yaml

```diff
@@ -74,34 +74,38 @@ jobs:
 
   update_network_operator_values:
     needs:
-    - determine_docker_registry_and_tag
-    - build_and_push_images
+      - determine_docker_registry_and_tag
+      - build_and_push_images
     runs-on: ubuntu-latest
     env:
+      DOCKER_REGISTRY: ${{ needs.determine_docker_registry_and_tag.outputs.docker_registry }}
       DOCKER_TAG: ${{ needs.determine_docker_registry_and_tag.outputs.docker_tag }}
       GH_TOKEN: ${{ secrets.GH_TOKEN_NVIDIA_CI_CD }}
     steps:
-    - uses: actions/checkout@v4
-      with:
-        token: ${{ secrets.GH_TOKEN_NVIDIA_CI_CD }}
-        repository: ${{ github.repository_owner }}/network-operator
-    - name: Create PR to update image tags in network-operator values
-      run: |
-        git config user.name  nvidia-ci-cd
-        git config user.email svc-cloud-orch-gh@nvidia.com
-
-        git checkout -b feature/update-sriov-tags-to-$DOCKER_TAG
-        sed -Ei "s/(version: )network-operator-.+/\\1$DOCKER_TAG/g" hack/release.yaml
-        make release-build
+      - uses: actions/checkout@v4
+        with:
+          token: ${{ secrets.GH_TOKEN_NVIDIA_CI_CD }}
+          repository: ${{ github.repository_owner }}/network-operator
+      - name: Determine base branch
+        run: |
+          echo "BASE_BRANCH=${{ contains(env.DOCKER_TAG, 'beta') && 'master' || env.DOCKER_TAG }}" >> $GITHUB_ENV
+      - name: Create PR to update image tags in network-operator values
+        run: |
+          git config user.name  nvidia-ci-cd
+          git config user.email svc-cloud-orch-gh@nvidia.com
+          
+          git checkout -b feature/update-sriov-tags-to-$DOCKER_TAG
+          yq -i e '.SriovNetworkOperator.repository |= "${{ env.DOCKER_REGISTRY }}"' hack/release.yaml
+          yq -i e '.SriovNetworkOperator.version |= "${{ env.DOCKER_TAG }}"' hack/release.yaml
```

Comment:
Some steps are different from current `draft-image-builder.jenkinsfile`
- Need to sync the helm chart
- When is the Network Operator image built and release.yaml updated with it?
- All GA handling (different repositories)

Reply:
> Some steps are different from current `draft-image-builder.jenkinsfile`
> 
> * Need to sync the helm chart
done
> * When is the Network Operator image built and release.yaml updated with it?
it will be done in network operator action
> * When is the Chart.yaml updated?
@rollandf  could you please add more details regarding this comment?
> * All GA handling (different repositories)
done

Reply:
Regarding chart.yaml, this is waht we have in `draft-image-builder.jenkinsfile`

                    def chart_file = './deployment/network-operator/Chart.yaml'
                    def ChartData = readYaml file: "${chart_file}"

                    ChartData.version = "${(version.substring(1))}"
                    ChartData.appVersion = "${version}"

                    writeYaml file: "${chart_file}", data: ChartData, overwrite: true

Reply:
@e0ne I am OK to merge this and iterate on follow ups

Reply:
> Regarding chart.yaml, this is waht we have in `draft-image-builder.jenkinsfile`

it should be fixed in `package-and-push-helm-chart` job or some other which merges code into the repo